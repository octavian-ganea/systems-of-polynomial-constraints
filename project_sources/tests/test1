% Test 2:
x^2 + y^2 - 1 > 0
x^3 - y^2 = 0
