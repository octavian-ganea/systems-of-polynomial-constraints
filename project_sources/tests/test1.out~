====================================================
Parsing file test1
Parsing succeded. Results:
List of all variables:
Vector(x, y)
List of all polynomials:
Vector((1.0 y^2 + -1.0 ) + 1.0 x^2, (-1.0 y^2) + 1.0 x^3)

List of desired signs : Vector(>, =)
Analysed polynomials: 
(1.0 y^2 + -1.0 ) + 1.0 x^2
(-1.0 y^2) + 1.0 x^3
vars: Vector(x, y)

Univariate projection (on real axis) contains these polynomials: 
1.0 y^2 + -1.0 
-1.0 y^2
4.0 y^2 + -4.0 
27.0 y^4
-1.0  + -2.0 y^4 + 3.0 y^2 + 1.0 y^6
1.0  + -1.0 y^2
-1.0  + 3.0 y^2 + 1.0 y^6 + -3.0 y^4
1.0 y^4

There are 61 cells
====== FINAL RESULT FOR test1 ================================
Map(x -> 1.0, y -> -1.0)

