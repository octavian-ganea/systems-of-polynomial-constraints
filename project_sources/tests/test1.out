====================================================
Parsing file test1
Parsing succeded. Results:
List of all variables:
Vector(x, y)
List of all polynomials:
Vector((1.0 y^2 + -1.0 ) + 1.0 x^2, (-1.0 y^2) + 1.0 x^3)

List of desired signs : Vector(>, =)
Analysed polynomials: 
(1.0 y^2 + -1.0 ) + 1.0 x^2
(-1.0 y^2) + 1.0 x^3
vars: Vector(x, y)

Univariate projection (on real axis) contains these polynomials: 
1.0 y^2 + -1.0 
-1.0 y^2
4.0 y^2 + -4.0 
27.0 y^4
-1.0  + -2.0 y^4 + 3.0 y^2 + 1.0 y^6
1.0  + -1.0 y^2
-1.0  + 3.0 y^2 + 1.0 y^6 + -3.0 y^4
1.0 y^4

There are 61 cells
Map(x -> 0.5874010516826402, y -> -2.0)
Map(x -> 1.5874010516826402, y -> -2.0)
Map(x -> 2.5874010516826402, y -> -2.0)
Map(x -> -1.0000010003753592, y -> -1.0)
Map(x -> -1.0003753592931224E-6, y -> -1.0)
Map(x -> 0.49999949981232034, y -> -1.0)
Map(x -> 1.0, y -> -1.0)
Map(x -> 2.0, y -> -1.0)
Map(x -> -1.5608273029584716, y -> -0.8279328090627981)
Map(x -> -0.5608273029584716, y -> -0.8279328090627981)
Map(x -> 5.4420523643017304E-11, y -> -0.8279328090627981)
Map(x -> 0.5608273030673127, y -> -0.8279328090627981)
Map(x -> 0.7212730120796835, y -> -0.8279328090627981)
Map(x -> 0.8817187210920543, y -> -0.8279328090627981)
Map(x -> 1.8817187210920543, y -> -0.8279328090627981)
Map(x -> -1.7548776664250498, y -> -0.6558656181255962)
Map(x -> -0.7548776664250498, y -> -0.6558656181255962)
Map(x -> -2.273231602956116E-11, y -> -0.6558656181255962)
Map(x -> 0.7548776663795852, y -> -0.6558656181255962)
Map(x -> 1.7548776663795853, y -> -0.6558656181255962)
Map(x -> -1.9447008758022881, y -> -0.32793330925047776)
Map(x -> -0.9447008758022882, y -> -0.32793330925047776)
Map(x -> -0.23457863055477796, y -> -0.32793330925047776)
Map(x -> 0.4755436146927323, y -> -0.32793330925047776)
Map(x -> 0.7101222452073828, y -> -0.32793330925047776)
Map(x -> 0.9447008757220333, y -> -0.32793330925047776)
Map(x -> 1.9447008757220332, y -> -0.32793330925047776)
Map(x -> -2.0, y -> -1.0003753592931224E-6)
Map(x -> -1.0, y -> -1.0003753592931224E-6)
Map(x -> -0.4999954555016004, y -> -1.0003753592931224E-6)
Map(x -> 9.088996799199279E-6, y -> -1.0003753592931224E-6)
Map(x -> 0.5000045444983996, y -> -1.0003753592931224E-6)
Map(x -> 1.0, y -> -1.0003753592931224E-6)
Map(x -> 2.0, y -> -1.0003753592931224E-6)
Map(x -> -1.9447012230281575, y -> 0.3279323088697413)
Map(x -> -0.9447012230281575, y -> 0.3279323088697413)
Map(x -> -0.2345792877823868, y -> 0.3279323088697413)
Map(x -> 0.4755426474633839, y -> 0.3279323088697413)
Map(x -> 0.7101219352234965, y -> 0.3279323088697413)
Map(x -> 0.9447012229836091, y -> 0.3279323088697413)
Map(x -> 1.9447012229836091, y -> 0.3279323088697413)
Map(x -> -1.7548776664272463, y -> 0.6558656181148419)
Map(x -> -0.7548776664272463, y -> 0.6558656181148419)
Map(x -> -2.716887825826575E-11, y -> 0.6558656181148419)
Map(x -> 0.7548776663729085, y -> 0.6558656181148419)
Map(x -> 1.7548776663729084, y -> 0.6558656181148419)
Map(x -> -1.5608273029592474, y -> 0.827932809057421)
Map(x -> -0.5608273029592474, y -> 0.827932809057421)
Map(x -> 5.586381357503001E-11, y -> 0.827932809057421)
Map(x -> 0.5608273030709751, y -> 0.827932809057421)
Map(x -> 0.7212730120793975, y -> 0.827932809057421)
Map(x -> 0.8817187210878199, y -> 0.827932809057421)
Map(x -> 1.88171872108782, y -> 0.827932809057421)
Map(x -> -1.0000010003753592, y -> 1.0)
Map(x -> -1.0003753592931224E-6, y -> 1.0)
Map(x -> 0.49999949981232034, y -> 1.0)
Map(x -> 1.0, y -> 1.0)
Map(x -> 2.0, y -> 1.0)
Map(x -> 0.5874010516826402, y -> 2.0)
Map(x -> 1.5874010516826402, y -> 2.0)
Map(x -> 2.5874010516826402, y -> 2.0)
====== FINAL RESULT FOR test1 ================================
Map(x -> 1.0, y -> -1.0)

